src/de/hhn/aib/labsw/blackmirror/view/widgets/ReminderWidget.java

```diff
@@ -52,127 +59,108 @@ private void initComponents() {
-                0, ZoneId.of("Europe/Berlin")), "Meetingnnnnnnnnnnnnnnnmnnnnnnnnnn");
-        Event e2 = new Event(ZonedDateTime.of(2022, 8, 12, 10, 30, 0,
-                0, ZoneId.of("Europe/Berlin")), "Meeting");
-        events.add(e2);
-        events.add(e1);
-
-        events.sort(new EventComparator());
-        if (!events.isEmpty()) {
+        Reminder r1 = new Reminder(LocalDate.now(), "Daily Scrum um 14 Uhr");
+        Reminder r2 = new Reminder(LocalDate.now(), "Programmieren usw.");
+
+        reminders.add(r2);
+        reminders.add(r1);
+
+        c.gridy = 2;
+        if (!reminders.isEmpty()) {
             int i = 0;
-            List<String> eventsStr = new ArrayList<>();
-            for (Event ev :
-                    events) {
-                if (i < 6) {
-                    eventsStr.add(parseEvent(ev));
+            List<String> reminderStr = new ArrayList<>();
+            for (Reminder r :
+                    reminders) {
+                if (i < 8) {
+                    reminderStr.add(parseReminder(r));
                     i++;
                 }
             }
 
-            JList list = new JList(eventsStr.toArray());
+            if (i==6) {
+                reminderStr.add("...");
+            }
+
+            JList list = new JList(reminderStr.toArray());
             list.setBackground(Color.BLACK);
             list.setForeground(Color.WHITE);
+            list.setFont(new Font(list.getFont().getName(), list.getFont().getStyle(), 13));
 
-            panel.add(list, BorderLayout.NORTH);
+            panelMain.add(list, c);
         } else {
-            JLabel noEventsText = new JLabel(resources.getString("noEvents"));
-            noEventsText.setBackground(Color.BLACK);
-            noEventsText.setForeground(Color.WHITE);
+            JLabel noRemindersText = new JLabel(resources.getString("noReminders"));
+            noRemindersText.setBackground(Color.BLACK);
+            noRemindersText.setForeground(Color.WHITE);
 
-            panel.add(noEventsText);
+            panelMain.add(noRemindersText, c);
         }
-        panel.setBackground(Color.BLACK);
-
-        JLabel iconLabel = new JLabel("", SwingConstants.CENTER);
-        // icon author
-        ImageIcon remindIcon = new ImageIcon("icons/reminder_icon.jpg");
-        iconLabel.setIcon(remindIcon);
-
-        reminderLabel.setVerticalAlignment(SwingConstants.NORTH);
-
-
-        panelMain.add(iconLabel);
-        panelMain.add(reminderLabel);
-        panelMain.add(panel);
-        panelMain.setLayout(new GridLayout(3,1));
         this.add(panelMain);
-
-        // SwingUtils.setFont(this, new Font("Calibri", Font.PLAIN, 18));
     }
 
     /**
-     * Requests all Events and adds them to a list if they are on the same date.
+     * Requests all Reminders and adds them to a list if they are on the same date.
      */
-    private void getTodayEvents() {
-        ArrayList<Event> requestedEvents = new ArrayList<>();
```

Comment:
Warum nutzt du hier ArrayList und weiter oben List?
Einheitlich mit dem selben List-Typen wäre konsequenter

Reply:
hab das wohl übersehen, dass das eine noch eine List war. Sollte aber kein Problem sein

Reply:
Ansonsten alles in Ordnung von meiner Seite. Kannst von mir aus mergen 👍